config/crd/bases/xgboostjob.kubeflow.org_xgboostjobs.yaml

```diff
@@ -8,6 +8,19 @@ metadata:
   creationTimestamp: null
   name: xgboostjobs.xgboostjob.kubeflow.org
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.conditions[-1:].type
+    name: State
+    type: string
+  - JSONPath: .metadata.creationTimestamp
+    name: Age
+    type: date
+  - JSONPath: .spec.ttlSecondsAfterFinished
+    name: TTL-Seconds
```

Comment:
use full name is better, TTL-After-Finished-Seconds

Reply:
done